gwf/colours.py

```diff
@@ -0,0 +1,10 @@
+COLORS = {
+    'red': '\033[91m',
+    'blue': '\033[94m',
+    'green': '\033[32m',
+    'black': '\033[30m',
+    'yellow': '\033[33m',
+    'bold': '\033[1m',
+}
+
+CLEAR = '\033[0m'
```

Comment:
The `colours.py` module has been completely removed (42f008b) since it is not cross-platform compatible (we should be using something like https://pypi.python.org/pypi/colorama instead).